Octokit.Reactive/Clients/IObservableRepositoryInvitationsClient.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Collections.Generic;
+
+namespace Octokit.Reactive
+{
+    public interface IObservableRepositoryInvitationsClient
+    {
+        /// <summary>
+        /// Accept a repository invitation.
+        /// </summary>
+        /// <remarks>
+        /// See the <a href="https://developer.github.com/v3/repos/invitations/#accept-a-repository-invitation">API documentation</a> for more information.
+        /// </remarks>        
+        /// <param name="invitationId">The id of the invitation.</param>        
+        IObservable<bool> Accept(int invitationId);
+
+        /// <summary>
+        /// Decline a repository invitation.
+        /// </summary>
+        /// <remarks>
+        /// See the <a href="https://developer.github.com/v3/repos/invitations/#decline-a-repository-invitation">API documentation</a> for more information.
+        /// </remarks>        
+        /// <param name="invitationId">The id of the invitation.</param>        
+        IObservable<bool> Decline(int invitationId);
+
+        /// <summary>
+        /// Deletes a repository invitation.
+        /// </summary>
+        /// <remarks>
+        /// See the <a href="https://developer.github.com/v3/repos/invitations/#delete-a-repository-invitation">API documentation</a> for more information.
+        /// </remarks>
+        /// <param name="repositoryId">The id ot the repository.</param>
+        /// <param name="invitationId">The id of the invitation.</param>        
+        IObservable<bool> Delete(int repositoryId, int invitationId);
+
+        /// <summary>
+        /// Gets all invitations for the current user.
+        /// </summary>
+        /// <remarks>
+        /// See the <a href="https://developer.github.com/v3/repos/invitations/#list-a-users-repository-invitations">API documentation</a> for more information.
+        /// </remarks>       
+        /// <returns>A <see cref="IReadOnlyList{RepositoryInvitation}"/> of <see cref="RepositoryInvitation"/>.</returns>
```

Comment:
remove `<returns>` line as it's incorrect (should be `Observable` not `IReadOnlyList`) but also it's not adding any value to simply say what the return type is, so just :fire: the whole line